test/gcm.provider.test.js

```diff
@@ -88,7 +88,7 @@ describe('GCM provider', function() {
 
   describe('for multiple device tokens', function() {
     it('sends Notification as a GCM message', function(done) {
-      var notification = aNotification({ aKey: 'a-value' });
+      var notification = aNotification({ aKey: 'a-value', aNull: null, message: undefined });
```

Comment:
Should have added better notes for this as well.  As part of the code style changes I changed the logic test here: https://github.com/clarkbw/loopback-component-push/commit/0f571dbd6c3c8701a90b9ef4813425c2f4094476#diff-8ba8b7926d5e215abcfc9cff520946b2L71 and even though its obvious I wanted to have the test show that non-truthy values were still ignored.